www/front_src/src/components/header/getResourcesUrl.ts

```diff
@@ -0,0 +1,65 @@
+const hostCriterias = { resourceTypes: [{ id: 'host', name: 'Host' }] };
+const serviceCriteria = { resourceTypes: [{ id: 'service', name: 'Host' }] };
+
+interface StatusCriterias {
+  statuses: Array<{ id: string; name: string }>;
+}
+
+const getStatusCriterias = (status): StatusCriterias => {
+  return { statuses: [status] };
+};
+
+const downCriterias = getStatusCriterias({ id: 'DOWN', name: 'Down' });
+const unreachableCriterias = getStatusCriterias({
+  id: 'UNREACHABLE',
+  name: 'Unreachable',
+});
+const upCriterias = getStatusCriterias({ id: 'UP', name: 'Up' });
+const pendingCriterias = getStatusCriterias({ id: 'PENDING' });
```

Comment:
no "name" in this object?

Reply:
Actually no name is required in any object, i'll remove them.